views/routes/routes.js

```diff
@@ -88,7 +88,7 @@ export const apiRoutes = {
   },
   chart: {
     show: (chart_id) => `${apiPath}/charts/${chart_id}`,
-    index: `${apiPath}/charts`,
+    all: `${apiPath}/charts`,
```

Comment:
why is this changing to `all`? The others use `index`. Also, do we need to tack on the query params here like we do elsewhere, such as participants above?

```js
  participants: {
    index: (queryParams) =>
      queryParams
        ? `${apiPath}/participants?${qs.stringify(queryParams)}`
        : `${apiPath}/participants`,
  },
```

Reply:
The get request handles this for us. I am doing work with the participants search, this looks like an opportunity to update this code.
https://github.com/AMP-SCZ/dpdash/blob/a29e5969c31f75811c17837913ec76d64252f418/views/api/http.js#L29-L40